.github/workflows/update-generated-files.yaml

```diff
@@ -9,7 +9,7 @@ env:
   BRANCH_NAME: sync-generated-files
 
 jobs:
-  update_genereated_files:
```

Comment:
I didn't see any references in the codebase with the current job name, `update_genereated_files`.